en/tutorials/linux_sbus.md

```diff
@@ -0,0 +1,35 @@
+# Sbus Driver for linux（drivers/linux_sbus）
+This driver allows the autopilot to fetch data from the futaba sbus receiver via serial port, up to 16 channels.  
```

Comment:
Other receivers based on the sbus protocol are available, such as frsky, radioLink, and even sbus encoders.